flink-connector-debezium/src/main/java/com/ververica/cdc/debezium/DebeziumSourceFunction.java

```diff
@@ -72,9 +73,6 @@
 import java.util.concurrent.ThreadFactory;
 import java.util.concurrent.TimeUnit;
 
-import static com.alibaba.ververica.cdc.debezium.utils.DatabaseHistoryUtil.registerHistory;
-import static com.alibaba.ververica.cdc.debezium.utils.DatabaseHistoryUtil.retrieveHistory;
```

Comment:
It seems many the static import is lost, is that on purpose?

Reply:
No, I'll update all static imports